etc/db_schema_filemetadata_table.sql

```diff
@@ -0,0 +1,13 @@
+/*
+ * Feckless Weasel SQL Installer FileMetadata Table Script
+ */
+
+CREATE TABLE FileMetadata(
+	fid INT AUTO_INCREMENT,
+	user varchar(25) NOT NULL,
```

Comment:
We should replace user with uid, the INT field from the user table that identifies the user and use a foreign key constraint.

In addition, it might be worth coming up with a strategy for when users are deleted. Should we ON DELETE SET NULL and orphan the file?